master_slave_pose_mapping:
  # Slave robot command mode(absolute/incremental).
  slave_command_mode: incremental # default: incremental
  # Slave robot command content(position/orientation/pose)
  slave_command_content: pose # default: pose
  # Name of command frame id for slave robot.
  slave_command_frame_id: panda_hand_tcp # default: panda_EE
  # Command pose mapping coordinate order of master robot relative to slave robot([x,y,z,rx,ry,rz]).
  pose_mapping_order: "['z', 'x', 'y', 'rx', 'ry', 'rz']" # default: "['x', 'y', 'z', 'rx', 'ry', 'rz']"
  # Absolute workspace scale of master robot relative to slave robot([x,y,z,rx,ry,rz]).
  absolute_workspace_scale: "[4.05, 6.25, 7.125, -1, -1, 1]" # default: "[4.05, 6.25, 7.125, 1, 1, 1]"
  # Incremental workspace scale of master robot relative to slave robot([x,y,z,rx,ry,rz]).
  incremental_workspace_scale: "[0.5, 0.5, 0.5, -0.25, -0.25, 0.25]" # default: "[1, 1, 1, 1, 1, 1]"
